mininterface/tk_interface/tk_window.py

```diff
@@ -31,6 +30,8 @@ def __init__(self, interface: "TkInterface"):
         self.interface = interface
         self.title(interface.title)
         self.bind('<Escape>', lambda _: self._ok(Cancelled))
+        self.bind('<Control-h>', self._show_help)  # Help with Ctrl+H
```

Comment:
now, a help shows the hint for secret even if no secret is on the page. I prefer not having such help :)